projects/pastanaga-angular/src/lib/controls/form-field.utils.ts

```diff
@@ -66,3 +66,10 @@ export function sanitizeStringValue(value: any, acceptHtmlTags: boolean) {
     }
     return value;
 }
+
+export function sanitizeStringValue2(value: any) {
```

Comment:
we could just use the existing `sanitizeStringValue` by making its `acceptHtmlTags` parameter optional, no?

Reply:
yes... this was temporary, removed in a latter commit